### PR TITLE
404Page

### PR DESCRIPTION
**Pull request by padmajabhol**
Screenshot (88)
Changed UI of the page.

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
could be better if related to our project

**Comment by padmajabhol** [comment]
> could be better if related to our project

By changing the neon green to the color?

**Comment by smaranjitghose** [comment]
404 page is not loading. Check the deploy [preview](https://deploy-preview-808--doc2pen.netlify.app/404) please

File: src/pages/404/notFound.module.css [hunk 1]
```diff
@@ -3,7 +3,9 @@
 .root {
   font-family: "Amatic SC", sans-serif;
   font-size: 100px;
-  color: #0b3c54;
+
```
Comment (smaranjitghose) [inline src/pages/404/notFound.module.css:6]: Avoid adding extra whitespaces between lines

**Comment by smaranjitghose** [comment]
Use react-spring to make a custom animation.

There should be a button to go back to Home Page as well

**Comment by smaranjitghose** [comment]
> > Use react-spring to make a custom animation.
> > There should be a button to go back to Home Page as well
> 
> Please provide me a little more time so I can redesign the page as per your web app's theme.

Sure. Take 2 more days. If required, we can extend further

**Comment by smaranjitghose** [comment]
Closed due to inactivity

**Pull request closed without merging**